lib/user.rb

```diff
@@ -0,0 +1,29 @@
+require 'pry'
+
+
+class User
+
+attr_reader :name, :jokes
+
+  def initialize(name)
+    @name = name
+    @jokes = []
+  end
+
+  def learn(joke)
+    @jokes << joke
+  end
+
+  def tell(who, what)
+    who.jokes << what
```

Comment:
nice! who is a user instance, so you can call `.jokes` on it, and `what` is a joke, so this works.  You can also use your `learn` method that you wrote above, since you took the trouble to write it, and it does the same thing 🙂